Visualizer/src/stores/simulation.js

```diff
@@ -1,6 +1,6 @@
 import { defineStore } from "pinia";
 import { useStorage } from "@vueuse/core";
-import { emitAgentsSelectedEvent, emitTickEvent } from "../scripts/emitter";
+import { emitAgentsSelectedEvent, emitTickEvent } from "@/scripts/emitter";
```

Comment:
use relative paths

Reply:
WebStorm 2022.2.1RC always changes it back to absolute paths. I think it does not really matter, but we should freeze it in the linting rules.